plugins/luxtronic2/__init__.py

```diff
@@ -328,22 +399,44 @@ def _decode(self, identifier, value):
         return value
 
     def parse_item(self, item):
+        def __reverseListOp(param):
+            # reversing list building of smarthome.py config parser for the
+            # binary or opperator |
```

Comment:
I think putting the value in quotes will fix this problem too. Usually you should be using quotes around the value to avoid escaping problem.

Other setting may have this problem too, but we should not start to implement such an "unquoting mechanism" for each of these settings. A general solution, which already exists (using quotes), would be more clear and consistent.